### DISCUSSION TITLE
微信交互菜单配置[Discussion]:

### DISCUSSION THREAD

**Discussion by youyifentian** [jxxghp/MoviePilot#378] [讨论]
### 项目讨论

没有找到关于微信交互的菜单布局配置，请大佬给个菜单配置截图或者教程吧

**Comment by mjsoftking** [comment]
请教一下，怎么过的域名认证

**Reply by suble** [reply]
我有腾讯云挂了网站。我就在宝塔上做个反代，就有固定IP了。

**Comment by LoSiTeE** [comment]
同求，不过MP现在功能不多，bug很多，还是初期阶段。wiki好像也没精力写
